test/EventStore.Client.Tests/ConnectionStringTests.cs

```diff
@@ -90,6 +94,11 @@ public class ConnectionStringTests {
 		static string MockingTone(string key) => new(key.Select((c, i) => i % 2 == 0 ? char.ToUpper(c) : char.ToLower(c)).ToArray());
 	}
 
+	public static IEnumerable<object?[]> InvalidClientCertificates() {
```

Comment:
Would it be worthwhile moving this closer to the test?

Reply:
Done